### DISCUSSION TITLE
Is there another emulatorjs docker container that isn't alpine based?

### DISCUSSION THREAD

**Discussion by stayupthetree** [RandomNinjaAtk/arr-scripts#89] [Q&A] [answered]
Was looking to setup the ra-rom-downloader, but looking at the setup.bash it has debian based apt commands and the only emulatorjs docker I see is from linuxserver that is alpine based and uses apk instead of apt. I cant seem to find one that is debian based

**Comment by RandomNinjaAtk** [comment]
I currently use this base image with it in my own setup: lsiobase/ubuntu:focal

Keep in mind that as of now, there isn't really any documentation, and its something I've been experimenting with.... So your mileage may vary... 

Your welcome to give it a try...

**Reply by stayupthetree** [reply]
Yeah probably too much to try and sort out right now and figure it out in Unraid. Just tinkering around. May give it a go down the road. Thanks for the reply!!

**Reply by RandomNinjaAtk** [reply, marked as answer]
You just go to the docker page and click add a container and fill it out like the image below:

![Screenshot from 2023-09-26 21-47-29](https://github.com/RandomNinjaAtk/arr-scripts/assets/8386416/56b180bd-ea7d-4fd2-9035-cbbc0270dc4a)


You add the init script, just like you would for the other containers to the correct folder.
